<launch>
    <arg name="host" default="localhost"/>
    <arg name="port" default="2000"/>
    <arg name="timeout" default="300"/>

    <arg name="town" default="Town13"/>
    <arg name="role_name" default="hero"/>
    <arg name="objects_definition_file" default="/home/ailab/carla-ros-bridge/catkin_ws/src/ros-bridge/carla_spawn_objects/config/objects.json"/>

    <param name="use_sim_time" value="True"/>

    <include file="/home/ailab/carla-ros-bridge/catkin_ws/src/ros-bridge/carla_spawn_objects/launch/carla_spawn_objects.launch">
        <arg name="objects_definition_file" value="$(arg objects_definition_file)"/>
        <arg name="spawn_point_ego_vehicle" value=""/>
        <arg name="spawn_sensors_only" value="True"/>
    </include>

    <!--<node pkg="team2_package" type="carla_route_saver.py" name="carla_route_saver" output="screen"/>-->
    <!--<node pkg="team2_package" type="pseudo_controller" name="pseudo_controller" output="screen"/>-->

    <!-- <node pkg="team2_package" type="localizer" name="localizer" output="screen"/> -->
    <!-- <node pkg="team2_package" type="global_planner" name="global_planner" output="screen"/> -->
    <!-- <node pkg="team2_package" type="controller" name="controller" output="screen"/> -->
    <!-- <node pkg="team2_package" type="behavior_planner" name="behavior_planner" output="screen"/> -->

    <!-- <include file="/home/ailab/Desktop/CARLA_Leaderboard_2.0/CARLA_Leaderboard_20/team_code/catkin_ws/src/team2_package/launch/detector.launch"/> -->

    <node name="rviz" pkg="rviz" type="rviz" args="-d /home/ailab/Desktop/CARLA_Leaderboard_2.0/CARLA_Leaderboard_20/team_code/catkin_ws/src/team2_package/config/visualizer.rviz"/>

</launch>
